tokio/src/signal/windows.rs

```diff
@@ -135,6 +131,112 @@ unsafe extern "system" fn handler(ty: DWORD) -> BOOL {
     }
 }
 
+/// Creates a new stream which receives "ctrl-c" notifications sent to the
+/// process.
+///
+/// # Examples
+///
+/// ```rust,no_run
+/// use tokio::signal::windows::ctrl_c;
+///
+/// #[tokio::main]
+/// async fn main() -> Result<(), Box<dyn std::error::Error>> {
+///     // An infinite stream of CTRL-C events.
+///     let mut stream = ctrl_c()?;
+///
+///     // Print whenever a CTRL-C event is received.
+///     loop {
+///         stream.recv().await;
+///         println!("got signal CTRL-C");
+///     }
```

Comment:
This is a good example but with an infinite loop I think it may not be possible to kill it conventionally 😅 

Maybe it's worth changing this to a limited loop that exits after 3 ctrl-c events?

Reply:
Done. Good point :laughing: